libvast/src/system/archive.cpp

```diff
@@ -83,6 +48,67 @@ void archive_state::send_report() {
   self->send(accountant, std::move(r));
 }
 
+std::unique_ptr<segment_store::lookup> next_session(archive_state& state) {
```

Comment:
Please make this a member function of the archive state and document it accordingly.